master-detail-demo/src_hd/com/axonivy/demo/masterdetail/ProductList/ProductDetail.xhtml

```diff
@@ -0,0 +1,123 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/frame-10.xhtml">
+	<ui:define name="title">Product Detail </ui:define>
+	<ui:define name="content">
+		<script>
+			window.isHideTaskName = true;
+			window.isHideTaskAction = true;
+			window.isHideCaseInfo = true;
+		</script>
+			
+		<h:outputStylesheet name="layouts/styles/customStyle.css" />
+	<h:form id="addProductDialogForm" >
+		<div class="ui-g">
+			<div class="ui-g-4">
+				  <p:outputLabel  value="Product Detail" style="font-weight:bold;font-size:20px"/>

```

Comment:
Hi @fkalanti should we use the CMS for this plain text

Reply:
Yes, i replaced this fixed values with a CMS value.